haiku-apps/debugmonitor/debugmonitor-1.0.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A tiny but useful software for debugging"
+DESCRIPTION="DebugMonitor is a small but useful software that can improve \
+the reliability and maintainability of your code. This software comes with \
+a DebugMonitor and a DebugClient library. To use, simply include the client \
+library in your app's Makefile, rebuild your app, run DebugMonitor, and then \
+start up your app. The main window should be clickable, and once you do so, a \
+the DebugMonitor window will appear and should record actions performed."
+HOMEPAGE="https://github.com/HaikuArchives/DebugMonitor"
+COPYRIGHT="Alister Lee"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="10b9b52033ebf53c84d663d88d32eb115299d993"
+SOURCE_URI="https://github.com/HaikuArchives/DebugMonitor/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="cf3326ff118a4b6175c0aa7ecdb4fdf9e72496e0480aacd3a539fe107f126d92"
+SOURCE_FILENAME="DebugMonitor-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="DebugMonitor-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	DebugMonitor = $portVersion
+	app:DebugMonitor = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make -C DebugClient  OBJ_DIR=objects
+	make -C DebugMonitor OBJ_DIR=objects
+	make -C Scribble     OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/DebugMonitor/Examples \
+			 $appsDir/DebugMonitor/lib
+	cp Scribble/objects/Scribble $appsDir/DebugMonitor/Examples
+	cp DebugMonitor/objects/DebugMonitor $appsDir/DebugMonitor
+	cp DebugClient/objects/DebugClient.a $appsDir/DebugMonitor/lib
```

Comment:
I don't think we need to copy this archive over, since it appears to only be used by DebugMonitor and has already been statically linked to it.